### ISSUE TITLE
Does Dhall render times that are in timezones on the western hemisphere correctly?

### ISSUE THREAD

**Issue by isaacsanders** [dhall-lang/dhall-lang#1267]
I entered the following time `04:00:00-04:00` in the Dhall front page, and it rendered `"04:00:00--4:00"`.

The double `-` doesn't seem correct to me, but I could be wrong.

**Comment by Gabriel439** [comment]
Yeah, this is a bug in the Haskell implementation of Dhall (which is what the live demo on the website is based on).  I'm migrating this issue to the corresponding repository
